corelight/searchlibrary/c8db7f5d-e8b4-4c58-81fa-baeb4ec76858.query

```diff
@@ -1,4 +1,4 @@
 tag=corelight_x509 ax
-| eval (TIMESTAMP < time($(certificate.not_valid_before)) || TIMESTAMP > time($(certificate.not_valid_after)))
-| time "certificate.not_valid_after" expired_on
-| table "certificate.subject" "certificate.issuer" "san.dns" expired_on
+| eval (if (TIMESTAMP < time($(certificate.not_valid_before))) {Validity="NOT_YET_VALID";} if (TIMESTAMP > time($(certificate.not_valid_after))) {Validity="EXPIRED";})
```

Comment:
```
eval (if (TIMESTAMP < time($(certificate.not_valid_before))) {Validity="NOT_YET_VALID";} if (TIMESTAMP > time($(certificate.not_valid_after))) {Validity="EXPIRED";})
| require Validity
```
This can be simplified to
```
eval (
if (TIMESTAMP < time($(certificate.not_valid_before))) {
   validity="NOT_YET_VALID";
   return true;
} else if (TIMESTAMP > time($(certificate.not_valid_after))) {
   validity="EXPIRED";
   return true;
 } else {
    return false;
  }
 )
```